data-raw/lung_surv.R

```diff
@@ -0,0 +1,59 @@
+library(tidymodels)
+library(censored)
+
+data(cancer)
+
+lung <- lung %>% drop_na()
+lung_train <- lung[-c(1:50), ]
+lung_test <- lung[1:50, ]
+
+sr_spec <-
+  survival_reg(dist = "weibull") %>%
+  set_engine("survival") %>%
+  set_mode("censored regression")
+
+set.seed(1)
+sr_fit <- sr_spec %>% fit(Surv(time, status) ~ ., data = lung_train)
+
+censor_probs <- function(x) {
+  # JFC prodlim check for the right class by inspecting the call :-O
+  dat <- data.frame(time = x[, "time"], status = x[, "status"])
+  # TODO check for 2+ censored values
```

Comment:
Does this TODO need to be addressed in this PR, or will come later?

Reply:
We'll need to change this later using the code that is in tune. I would leave as-is for now and I'll add it in another PR.